<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>963</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Selection of network with a specified range of vessel diameters</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButtonInFile">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Amira network file</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelInFile">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>0</y>
      <width>391</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelOutFile">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>391</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonOutFile">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Output file</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonBrancher">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>260</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Go</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_result">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>310</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Result:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelResult">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>310</y>
      <width>491</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>10</y>
      <width>451</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_artery_vertex">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Main artery node #</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditArteryVertex">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>120</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>9165</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_vertex2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>100</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>(1-based)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditRatio">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>190</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_ratio">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Branch ratio limit</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditVeinVertex">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>150</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>7563</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_vein_vertex">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Main vein node #</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_RGB">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>103</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>R           G           B</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_aR">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>120</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>1.0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_aG">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>120</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0.0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_aB">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>120</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0.0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_vG">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>150</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0.0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_vB">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>150</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>1.0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_vR">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>150</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0.0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_use_branch_count">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>190</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Use branch count</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>963</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonInFile</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>inFileSelecter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>45</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel">
     <x>232</x>
     <y>44</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonOutFile</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>outFileSelecter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>18</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>231</x>
     <y>73</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonBrancher</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>brancher()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>278</x>
     <y>306</y>
    </hint>
    <hint type="destinationlabel">
     <x>382</x>
     <y>331</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>inFileSelecter()</slot>
  <slot>outFileSelecter()</slot>
  <slot>brancher()</slot>
 </slots>
</ui>
